client/reconfiguration/src/poll_based_state_client.cpp

```diff
@@ -29,7 +29,14 @@ concord::messages::ReconfigurationResponse PollBasedStateClient::sendReconfigura
   concord::messages::ReconfigurationResponse rres;
   try {
     if (read_request) {
-      bft::client::ReadConfig read_config{request_config, bft::client::LinearizableQuorum{}};
+      // TODO: State transfer can work with f + 1 as long as there are no byzantine replicas
```

Comment:
Most readers won't understand this comment.
Can you add that the poll_based_state_client is also used by state transfer as an async reconfiguration update mechanism?